docs/deferred-account-flow.md

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Deferred account creation flow (not launched)
+# Deferred account creation flow
```

Comment:
Looks great, but could you please also update the related clauses in:

- https://github.com/subscriptions-project/swg-js/blob/master/docs/flows.md
- https://github.com/subscriptions-project/swg-js/blob/master/src/api/subscriptions.js#L107

Reply:
Sure.